### PR TITLE
Add a quote-of-the-day feature

### PR DESCRIPTION
**Pull request by tonybaloney**
e.g.

```
$ pybible --qotd
"For which things' sake the wrath of God cometh on the children of disobedience:" - Colossians 3:6 (King James Bible)
```

### CODE REVIEW

File: pybible/__main__.py [hunk 1]
```diff
@@ -15,6 +16,8 @@ def configure_arg_parser() -> argparse.ArgumentParser:
                                  action="store_true")
     reference_group.add_argument("-nt", "--new_testament", help="Reference the new testament",
                                  action="store_true")
+    reference_group.add_argument("--qotd", help="Quote a single, random verse",
```
Comment (vitorpio) [inline pybible/__main__.py]: two argument options:
"-qotd"
"--quote_of_the_day"

File: pybible/__main__.py [hunk 2]
```diff
@@ -71,6 +74,12 @@ def process_arguments(parser: argparse.ArgumentParser):
         else:
             references = [Reference(verse.text, book.title, chapter.number, verse.number)]
 
+    elif args.qotd:
+        book = random.choice(bible)
+        chapter = random.choice(book.chapters)
+        verse = random.choice(chapter.verses)
+        return f"\"{verse.text}\" - {book.title} {chapter.number}:{verse.number} ({bible.name})"
```
Comment (vitorpio) [inline pybible/__main__.py]: replace with:
`references = [Reference(verse.text, book.title, chapter.number, verse.number)]`
too keep the new parameter the same as the others.

File: README.md [hunk 3]
```diff
@@ -51,23 +51,29 @@ Reference the fourth verse of the third chapter from the book of John:
 
     $ pybible --verse john 3 4
     Nicodemus saith unto him, How can a man be born when he is old? can he enter the second time into his mother's womb, and be born?
-    
-It is possible to add extra parameters for adding the reference on the output(-r, --reference) and count the size(-s, --size) of what is been referenced.
+
+Get a random verse with the `--qotd` arugmnet:
+
+    $ pybible --motd
```
Comment (vitorpio) [inline README.md]: Replace with --qotd

**Review by vitorpio — CHANGES_REQUESTED** [review]
Simple changes

**Comment by tonybaloney** [comment]
@vitorpio made the changes to your request

**Pull request merged**